### PR TITLE
Update release/v0.7 to v0.6 syntax

### PR DESCRIPTION
**Pull request by guanghaolow**

### CODE REVIEW

File: Chemistry/src/Runtime/JordanWigner/JordanWignerClusterOperatorEvolutionSet.qs [hunk 1]
```diff
@@ -60,7 +62,7 @@ namespace Microsoft.Quantum.Chemistry.JordanWigner {
         for(idx in 0..Length(idxFermions)-1){
```
Comment (cgranade) [inline Chemistry/src/Runtime/JordanWigner/JordanWignerClusterOperatorEvolutionSet.qs]: This can now be replaced with `idx in IndexRange(idxFermions)`.

Comment (anpaz-msft) [reply Chemistry/src/Runtime/JordanWigner/JordanWignerClusterOperatorEvolutionSet.qs]: done.

**Comment by anpaz-msft** [comment]
The [WARNING] are all existing from 0.6, so I'll send a different PR to fix them. We'll fix the `SystemTests` later.

**Comment by guanghaolow** [comment]
@anpaz-msft Some of the `SystemTests` use phase estimation and are probabilistic, which explains the random failures. How does one go about fixing a seed for each test?

**Comment by anpaz-msft** [comment]
@guanghaolow, we run the tests using a seed for the rng, so that way the test results become deterministic. Because the tests kept failing, I've added the seed to the LiH tests: see [commit f9cc661](https://github.com/microsoft/QuantumLibraries/pull/100/commits/f9cc661fb833fcc9cd6998c83c14ee5fdd03b6bd)

**Pull request merged**